config_gen.py

```diff
@@ -286,6 +305,8 @@ def run(cmd, *args, **kwargs):
         print("Running qmake in '{}' with Qt {}...".format(build_dir, qt_version))
         run(["qmake"] + configure_opts + [pro_files[0]], env=env_config,
             **proc_opts)
+        if os.uname()[0] == "Darwin":
```

Comment:
Well, it works under Linux without it, so it's probably safer to leave it in. I'm not entirely certain why the Makefile has a hardcoded compiler in it, which bothers me, but qmake has a Mac specific configuration/mkspec anyway, so this problem is unlikely to affect users of other platforms.

I think we should be providing a message to the user though, to make them aware of this step - something like:
```
print("Patching generated makefile...")
```
should suffice.